view/frontend/templates/index.phtml

```diff
@@ -1 +1,45 @@
-<h2>RP AUCTION INDEX</h2>
\ No newline at end of file
+<?php
+$viewModel = $block->getData("view-model");
+$auctionItems = $viewModel->getAllAuctionItems();
+?>
+<h2>Auction Items</h2>
+<table class="table" id="contact-table">
+    <thead>
+    <tr>
+        <th scope="col" class="col"><?= __('ID'); ?></th>
+        <th scope="col" class="col"><?= __('Item Name'); ?></th>
+        <th scope="col" class="col"><?= __('Item Sku'); ?></th>
+        <th scope="col" class="col"><?= __('Item Base Auction Price'); ?></th>
+        <th scope="col" class="col"><?= __('Item Owner Email'); ?></th>
+        <th scope="col" class="col"><?= __('Created At'); ?></th>
+        <th scope="col" class="col"><?= __('Modified At'); ?></th>
+    </tr>
+    </thead>
+    <tbody>
+    <?php foreach ($auctionItems as $auctionItem):?>
+        <tr>
+            <td data-th="ID" class="col">
+                <?php echo $auctionItem->getEntityId() ?>
```

Comment:
Using `<?php echo .... ?>` is fine. You can also use the `<?= .... ?>` shorthand though